### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by acolombo1**
Add index.html and style.css. Correct linter errors.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* Intended for h1 elements within the myh1 class */
+.myh1 {
+  margin: 25px;
+  background-color: rgb(240, 240, 240);
+  color: rgb(38, 95, 62);
+  font-family: arial, sans-serif;
+}
```
Comment (gemmen29) [inline style.css:7]: [OPTIONAL] Kindly make this information as a hyperlink, you can do that by surrounding username with [] then put the link between ().
```
[username](url)
```

**Review by gemmen29 — APPROVED** [review]
Hi @acolombo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (acolombo1) [reply style.css:7]: Thank you for your review! Still, I don't understand the Optional, where should I put the link? (inside the style.css?)

Comment (gemmen29) [reply style.css:7]: No, for the README links that contain your information like GitHub, Twitter and LinkedIn.

Comment (gemmen29) [reply style.css:7]: This part
image

**Pull request merged**